src/components/Badges/proptypes.js

```diff
@@ -1,8 +1,9 @@
 import PropTypes from 'prop-types';
 
 export const badgePropTypes = {
-  title: PropTypes.string.isRequired,
-  step: PropTypes.number,
+  text: PropTypes.string.isRequired,
+  url: PropTypes.string,
+  symbol: PropTypes.node,
```

Comment:
Interesting pattern colocating the PropTypes with the component as oppose to in a shared top level PropTypes folder.

Does the import path become unwieldy when a component needs to access another distant component's PropTypes?

Reply:
Yikes! probably would get unwieldy if we had a lot of components elsewhere that's using these components, but we don't. If we do have another component elsewhere that's needing badge prop-types, we can lean on full path imports so it shouldn't be too bad. But really, if there's another component that's composed of Badges, I'd prefer it to live in this Badges folder unless it's super unrelated.